Capstone/Autocorrect/AutocorrectFunctions.h

```diff
@@ -0,0 +1,17 @@
+//
+//  AutocorrectFunctions.h
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/14/22.
+//
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+@interface AutocorrectFunctions : NSObject
++ (instancetype)shared;
+@property (strong, nonatomic) NSMutableDictionary *keyCoordinates;
+- (NSString *)findCorrectWord:(NSString *)searchWord :(NSArray *)colleges;
```

Comment:
colons after parameters are a bit wonky— let's keep the parameter naming, even if there is some degree of repetition
```suggestion
- (NSString *)findCorrectWord:(NSString *)searchWord colleges:(NSArray *)colleges;
```